Apps/Playground/Scripts/experience.js

```diff
@@ -27,7 +27,7 @@ function CreateSpheresAsync() {
     for (var i = 0; i < size; i++) {
         for (var j = 0; j < size; j++) {
             for (var k = 0; k < size; k++) {
-                var sphere = BABYLON.Mesh.CreateSphere("sphere" + i + j + k, 32, 0.9, scene);
+                var sphere = BABYLON.Mesh.CreateSphere("sphere" + i + j + k, 32, 0.9);
```

Comment:
Would it not be better to pass in the scene to `CreateSpheresAsync` instead?

Reply:
If we do that, then we should also do it to CreateBoxAsync.

Reply:
I'll add this change to script